gems/sorbet-runtime/lib/types/configuration.rb

```diff
@@ -393,6 +393,30 @@ def self.legacy_t_enum_migration_mode?
     @legacy_t_enum_migration_mode || false
   end
 
+  # @param [Array] sealed_violation_whitelist An array of Regexp to validate
+  #   whether inheriting /including a sealed module outside the defining module
+  #   should be allowed. Useful to whitelist benign violations, like shim files
+  #   generated for an autoloader.
+  def self.sealed_violation_whitelist=(sealed_violation_whitelist)
+    case sealed_violation_whitelist
+    when NilClass then nil
+    when Array
+      sealed_violation_whitelist.each do |x|
+        case x
+        when Regexp then nil
+        else raise TypeError.new("sealed_violation_whitelist accepts an Array of Regexp")
+        end
+      end
+    else
+      raise TypeError.new("sealed_violation_whitelist= accepts an Array of Regexp")
+    end
+
+    @sealed_violation_whitelist = sealed_violation_whitelist
```

Comment:
Can we make it that it can't be changed after it was set once?

Reply:
I actually implemented that, and then realized that I needed a way to reset it to it's default value for the purpose of testing.